Dockerfile

```diff
@@ -4,7 +4,7 @@ FROM python:3.6
 RUN pip install --no-cache-dir --ignore-installed \
 		pytest \
         pytest-cov \
-        google-cloud-bigquery==0.24 \
+        google-cloud-bigquery \
```

Comment:
nechal bych zafixovanou verzi, na který jsi to vyvíjel, v budoucnu zase může vyjít nějaká nová, která to při buildu rozbije